transformations/americanize_britishize_english/README.md

```diff
@@ -0,0 +1,25 @@
+# British to American English and vice versa translation
+Author name: Aman Srivastava (amanit0812@gmail.com)
+
+## What type of transformation is this?
+This transformation takes a sentence and converts it from british english to american english and vice-versa. 
+It supports following scenarios:-
+* Sentences with British English words :arrow_right: To American English
+* Sentences with American English words :arrow_right: To British English
+* Sentences with both American English and British English words :arrow_right: Translates American English words to British English words and vice versa.
```

Comment:
Is "Translates" the right verb here'?

Reply:
Generally, the word translation is used when the context is two languages. American English and British English are considered as two separate entities hence the work "Translates". Any suggestions for a better verb are welcomed.